src/components/DocumentationTopic/PrimaryContent/Mentions.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2024 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <section v-if="topics.length > 0" class="mentions">
+    <h2 class="mentions" id="mentions">Mentioned in</h2>
+    <Mention v-for="(topic, i) in topics"
+      :key="i"
+      :url="topic.url"
+      :title="topic.title"
+      :role="topic.role"
+      :kind="topic.kind"
+    />
+  </section>
+</template>
+
+<script>
+import Mention from 'docc-render/components/DocumentationTopic/PrimaryContent/Mention.vue';
+import referencesProvider from 'docc-render/mixins/referencesProvider';
+
+export default {
+  name: 'MentionedIn',
+  components: {
+    Mention,
+  },
+  mixins: [referencesProvider],
+  props: {
+    mentions: {
+      type: Array,
+      required: false,
+    },
+  },
+  computed: {
+    topics() {
+      if (this.mentions && this.mentions.length) {
+        return this.mentions
+          .slice(0, 5)
```

Comment:
Does the compiler ever provide more than this many items? Just curious if this maximum is only being enforced here or at both levels. (It's fine leaving in either way)

Reply:
The compiler could provide more, so it was decided that this would be handled at the presentation layer. Could change in the future. Thoughts?

Reply:
Makes sense. I was just curious. Sounds good.